src/providers/postgres/qgspostgresconn.h

```diff
@@ -188,6 +188,15 @@ class QgsPostgresResult
 
 };
 
+class PoolPostgresConn
```

Comment:
Can you add documentation that explains the rationale for this class?

Reply:
done

Reply:
(and renamed to Pgs...)